packages/server/.gitignore

```diff
@@ -33,5 +33,7 @@ lerna-debug.log*
 !.vscode/launch.json
 !.vscode/extensions.json
 
-# Uploaded photos
-/uploads
\ No newline at end of file
+# Keep folder for testing purposes in the GitHub Actions Workflow,
+# but don't keep the files inside it except .gitkeep
+uploads/*
```

Comment:
Required for unit tests in GitHub Actions. We exclude all files besides .gitkeep to keep the folder present.